docs/en/authoring/page_urls.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Page URLs
+pagename: authoring-page-urls
+lang: en
+---
+
+# Page URLs
+
+Page URLs are generated automatically at build time based on each page's `pagename`.  This allows URLs to remain static and independent of content, even if the title or purpose of a particular page changes.
+
+## Page Titles in URLs
+
+To make URLs more human-readable, we append the page titles to the URL like so: `https://mydocumentation.mysite.com/78b64024/?title=Page+URLs`.
```

Comment:
do we have full control over the URL here? 

I am looking at our zendesk links - they are on the form:

```
https://support.shotgunsoftware.com/hc/en-us/articles/115000068574-Integrations-User-Guide
```

Could we do something similar, e.g. `https://mydocumentation.mysite.com/78b64024?Page-URLs` or `https://mydocumentation.mysite.com/78b64024-Page-URLs`?

Reply:
Added a sentence to the URLs page describing what can be changed without affecting what page is loaded.